pkg/doc/vc/crypto/crypto.go

```diff
@@ -269,7 +271,7 @@ func (c *Crypto) getSDJWTSignedCredential(
 
 	joseSigner := jws.NewSigner(signingKeyID, jwsAlgName, signer)
 
-	sdjwt, err := credential.MakeSDJWT(joseSigner, signingKeyID)
+	sdjwt, err := credential.MakeSDJWT(joseSigner, signingKeyID, verifiable.MakeSDJWTWithHash(digestHashAlgo))
```

Comment:
pass hash algo from issuer profile

Reply:
we aren't doing that? please add a TODO & I will create a task.

Reply:
We used to use a default hash algo SHA256 (because credential.MakeSDJWT() wasn't able to take hashAgo argument and to change this behavior). 
Then, thanks @Moopli he added the ability to pass algo argument https://github.com/hyperledger/aries-framework-go/pull/3521/files and I just take the hashAlgo from issuer profile and pass it to MakeSDJWT